multi-node/aws/pkg/config/config.go

```diff
@@ -243,6 +242,18 @@ type Config struct {
 	TLSConfig *CompactTLSAssets
 }
 
+//This functionality now requires AWS API access
+//Separated out from cluster.Config() so that we can only compact
+//assets on command=up
+func (c *Config) CompactTLSAssets(tlsConfig *RawTLSAssets) error {
```

Comment:
The function name is the same as the type. This could get confusing. It also doesn't need to be exported.